inheritance-mapper/src/main/java/com/iluwatar/InheritanceMapper/App.java

```diff
@@ -0,0 +1,44 @@
+package com.iluwatar.InheritanceMapper;
+
+import com.iluwatar.InheritanceMapper.ClassObject.Bowler;
+import com.iluwatar.InheritanceMapper.ClassObject.Cricketer;
+import com.iluwatar.InheritanceMapper.ClassObject.Footballer;
+
+
+
+
+/**Inheritance mapper pattern.
+ * map each instance of class base on the inheritance type chosen
+ * To choose inheritance type go to ./ClassObject/Player.java
+ */
```

Comment:
Please explain the pattern purpose shortly. Then explain how this example implements it.